### PR TITLE
bump version for v1.0.0, and minor fixes for future releases

### PR DESCRIPTION
**Pull request by mjs**
- bump version in setup.py
- Updated project description to mention writing 
- Fix up import in write.py example

### CODE REVIEW

**Review by gferraro — APPROVED** [review]
Looks good, how was it working with wrong import before?

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 from setuptools import setup, find_packages, Extension
 
 long_description = """\
-This is Python package provides for quick, easy parsing for Cacophony
-Project Thermal Video (CPTV) files. It works with Python 3 only.
+This is Python package provides for quick parsing and generation of
+Cacophony Project Thermal Video (CPTV) files. It works with Python
+3.5+ only.
```
Comment (gferraro) [inline setup.py:6]: is there still f strings? 3.6+?

**Review by gferraro — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (mjs) [reply setup.py:6]: Nope, I got rid of the f-string. We need Python 3.5 support because of the pi.

Comment (gferraro) [reply setup.py:6]: oh ye ofcourse

**Pull request merged**